Formula/cocoapods.rb

```diff
@@ -15,11 +15,13 @@ class Cocoapods < Formula
   end
 
   depends_on "pkg-config" => :build
-  depends_on "ruby" if Hardware::CPU.arm?
-
   uses_from_macos "libffi", since: :catalina
   uses_from_macos "ruby", since: :catalina
 
+  on_arm do
+    depends_on "ruby"
+  end
```

Comment:
I wonder if this can be expressed as an extra argument for `uses_from_macos`, just like `since:`.